vj4/server.py

```diff
@@ -15,6 +16,9 @@
 options.define('listen', default='http://127.0.0.1:8888', help='Server listening address.')
 options.define('prefork', default=1, help='Number of prefork workers.')
 options.define('syslog', default=False, help='Use syslog instead of stderr for logging.')
+options.define('listen_owner', default='www-data', help='Owner of the socket when server is listening to a unix socket.')
```

Comment:
These changes can be extracted to a new PR so that I can merge these instantly.

Reply:
OK, i'm going to cherry-pick this commit and create a new pr.